packages/sn-controls-react/src/viewcontrols/NewView.tsx

```diff
@@ -14,7 +14,7 @@ import { reactControlMapper } from '../ReactControlMapper'
  * Interface for NewView properties
  */
 export interface NewViewProps {
-  onSubmit?: (path: string, content: ContentType, contentTypeName: string) => void
+  onSubmit?: (path: string, content: ContentType, contentTypeName: string, contentTemplateName?: string) => void
```

Comment:
I wonder if it is really needed to pass the contentTypeName twice here?

Reply:
What if a user want to pass not the contenttype's name but a custom contenttemplate?

Reply:
Because we pass in the `schema.ContentTypeName`, the user should handle itself. I still think that passing this down in this component is not necessary.